src/styles/main.scss

```
@@ -1,3 +1,9 @@
-body {
-  margin: 0;
-}
+@import "./utils/variable";
+@import "./normalize";
+@import "./blocks/page";
```

```suggestion
@import "./normalize";

@import "./blocks/page";
```
I would suggest an empty line between utils and blocks 😅